piwik/piwik
# Added possibility to create tracker automatically via paq after piwik…

refs https://github.com/piwik/piwik/issues/10878

To test use eg 
```html
<html>
<head></head>
<body>
<script type="text/javascript" src="//apache.piwik/piwik.js"></script>
</body>
</html>
```

Then initialize tracker after document was loaded via `_paq.push`. You will see tracker will be created etc. Also if someone else uses `Piwik.addTracker` the `_paq` will be correctly restored to use the trackerproxy. Automated test is not possible for this.

Will issue a PR for Piwik 3 where we log a message and not create the tracker
